__tests__/app.test.js

```diff
@@ -183,8 +183,12 @@ describe("POST /api/articles/:article_id/comments", () => {
 			.expect(201)
 			.then(({ body }) => {
 				expect(body.comment).toMatchObject({
-					username: "rogersop",
+					article_id: expect.any(Number),
```

Comment:
For the `article_id`, is it fair to say we would always know exactly what the `article_id` of a comment added on the endpoint "/api/articles/1/comments" will be, rather than expecting any number?